internal/conf/validate.go

```diff
@@ -233,53 +234,76 @@ func UnredactSecrets(input string, raw conftypes.RawUnified) (string, error) {
 		return input, errors.Wrap(err, "parse old config")
 	}
 
-	oldSecrets := make(map[string]string, len(oldCfg.AuthProviders))
+	oldAuthProviderSecrets := make(map[string]string, len(oldCfg.AuthProviders))
 	for _, ap := range oldCfg.AuthProviders {
 		if ap.Openidconnect != nil {
-			oldSecrets[ap.Openidconnect.ClientID] = ap.Openidconnect.ClientSecret
+			oldAuthProviderSecrets[ap.Openidconnect.ClientID] = ap.Openidconnect.ClientSecret
 		}
 		if ap.Github != nil {
-			oldSecrets[ap.Github.ClientID] = ap.Github.ClientSecret
+			oldAuthProviderSecrets[ap.Github.ClientID] = ap.Github.ClientSecret
 		}
 		if ap.Gitlab != nil {
-			oldSecrets[ap.Gitlab.ClientID] = ap.Gitlab.ClientSecret
+			oldAuthProviderSecrets[ap.Gitlab.ClientID] = ap.Gitlab.ClientSecret
 		}
 		if ap.Bitbucketcloud != nil {
-			oldSecrets[ap.Bitbucketcloud.ClientKey] = ap.Bitbucketcloud.ClientSecret
+			oldAuthProviderSecrets[ap.Bitbucketcloud.ClientKey] = ap.Bitbucketcloud.ClientSecret
 		}
 		if ap.AzureDevOps != nil {
-			oldSecrets[ap.AzureDevOps.ClientID] = ap.AzureDevOps.ClientSecret
+			oldAuthProviderSecrets[ap.AzureDevOps.ClientID] = ap.AzureDevOps.ClientSecret
 		}
 	}
 
-	newCfg, err := ParseConfig(conftypes.RawUnified{
+	newAuthProviderCfg, err := ParseConfig(conftypes.RawUnified{
 		Site: input,
 	})
 	if err != nil {
 		return input, errors.Wrap(err, "parse new config")
 	}
-	for _, ap := range newCfg.AuthProviders {
+	for _, ap := range newAuthProviderCfg.AuthProviders {
 		if ap.Openidconnect != nil && ap.Openidconnect.ClientSecret == redactedSecret {
-			ap.Openidconnect.ClientSecret = oldSecrets[ap.Openidconnect.ClientID]
+			ap.Openidconnect.ClientSecret = oldAuthProviderSecrets[ap.Openidconnect.ClientID]
 		}
 		if ap.Github != nil && ap.Github.ClientSecret == redactedSecret {
-			ap.Github.ClientSecret = oldSecrets[ap.Github.ClientID]
+			ap.Github.ClientSecret = oldAuthProviderSecrets[ap.Github.ClientID]
 		}
 		if ap.Gitlab != nil && ap.Gitlab.ClientSecret == redactedSecret {
-			ap.Gitlab.ClientSecret = oldSecrets[ap.Gitlab.ClientID]
+			ap.Gitlab.ClientSecret = oldAuthProviderSecrets[ap.Gitlab.ClientID]
 		}
 		if ap.Bitbucketcloud != nil && ap.Bitbucketcloud.ClientSecret == redactedSecret {
-			ap.Bitbucketcloud.ClientSecret = oldSecrets[ap.Bitbucketcloud.ClientKey]
+			ap.Bitbucketcloud.ClientSecret = oldAuthProviderSecrets[ap.Bitbucketcloud.ClientKey]
 		}
 		if ap.AzureDevOps != nil && ap.AzureDevOps.ClientSecret == redactedSecret {
-			ap.AzureDevOps.ClientSecret = oldSecrets[ap.AzureDevOps.ClientID]
+			ap.AzureDevOps.ClientSecret = oldAuthProviderSecrets[ap.AzureDevOps.ClientID]
 		}
 	}
-	unredactedSite, err := jsonc.Edit(input, newCfg.AuthProviders, "auth.providers")
+
+	unredactedSite, err := jsonc.Edit(input, newAuthProviderCfg.AuthProviders, "auth.providers")
 	if err != nil {
 		return input, errors.Wrap(err, `unredact "auth.providers"`)
 	}
 
+	var oldObservabilitySecrets []string
```

Comment:
```suggestion
	var observabilitySecretsReplaceList []string
```

is more accurate IMO (it has both old and new 😂 )